src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: New runtime transition guide
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+Synthetic monitoring has a new runtime that affects your public monitors. By converting from the legacy runtime to the new runtime, you improve the backend environment that your public monitors execute from. New synthetic monitors default to using the new runtime. If you've created a synthetic monitor using the legacy runtime, there are extra [configuration steps](#runtime-convert).
```

Comment:
We were making some changes to "original" where legacy had been used in the UI, because we didn't want people still using the old version to feel like it was bad or on the verge of being end-of-lifed.

Reply:
In this case, we are OK calling the older runtimes legacy. They will be labelled as such in the UI and EOL efforts for these older runtimes will pick up later in the year.